frame/staking/src/pallet/mod.rs

```diff
@@ -124,6 +125,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominations: Get<u32>;
 
+		/// Era History Depth for the validator rewards to be claimed. 
+		/// This should never be decreased once set.  
```

Comment:
We can probably do a better doc here.

Reply:
I added more doc based on your feedback. Have a look again.